.devcontainer/Dockerfile

```diff
@@ -123,8 +123,4 @@ ENV CXX=${USE_CLANG:+"clang++"}
 ENV CC=${CC:-"gcc"}
 ENV CXX=${CXX:-"g++"}
 
-# Include project
-#ADD . /workspaces/cpp_starter_project
-#WORKDIR /workspaces/cpp_starter_project
-
 CMD ["/bin/bash"]
```

Comment:
???

Reply:
That was not valid anymore since now the project folder is cpp_boilerplate_project. We didn't use this anyway and I consider it a bad practice to copy it because you might lose changes if the container is removed